src/main/java/com/exasol/adapter/dynamodb/mapping/AbstractPropertyToColumnMapping.java

```diff
@@ -42,4 +42,29 @@ public final DocumentPathExpression getPathToSourceProperty() {
     public final LookupFailBehaviour getLookupFailBehaviour() {
         return this.lookupFailBehaviour;
     }
+
+    @Override
+    public boolean equals(final Object other) {
+        if (this == other) {
+            return true;
+        }
+        if (!(other instanceof AbstractPropertyToColumnMapping)) {
+            return false;
+        }
+        if (!super.equals(other)) {
+            return false;
+        }
```

Comment:
I am not really sure. Since the `PropertyToColumnMapping` also extends `ColumnMapping`, I would remove this super comparison and use `getExasolColumnName()` to compare all significant field instances.

Reply:
I think this would lead to code dublication. I think it's more elegant to have the comparisson in only one place. Is there a disadvantage with using the super?

Reply:
No not really. It is still correct. My suggestion was more personal preference.